api/webhooks/webhooks.py

```diff
@@ -74,6 +74,10 @@ def call_organisation_webhooks(organisation, data, event_type):
     )
 
 
+def call_integration_webhook(config, data):
+    return _call_webhook(config, data)
```

Comment:
I appreciate the attempt to reuse the webhook functionality here but I'm not sure it's correct, right? The `_call_webhook` function expects a 'webhook' argument which inherits from `AbstractBaseWebhookModel`.

Reply:
Nope, WebhookConfiguration inherits from `AbstractBaseWebhookModel` https://github.com/Flagsmith/flagsmith/pull/393/files#diff-c2f900907b02278b20496d66a689e7b7b21abb20fc46c850417652e7a8eddaf8R7

Reply:
So it does. My bad.